superduperdb/ext/llm/openai.py

```diff
@@ -0,0 +1,19 @@
+import dataclasses as dc
+
+from superduperdb.ext.llm.base import BaseOpenAI
+
+
+@dc.dataclass
+class OpenAI(BaseOpenAI):
```

Comment:
Question is how to synchronize this with the other models from OpenAI. I.e. what 
about `ext/openai/model/*`?

Reply:
In the LLM scenario, I suggest guiding users to use the llm module in the documentation. For example, if a user wants to build a RAG application, they would first go to the llm directory to find the corresponding llm interface, and to the Embedding directory to find the embedding interface, instead of looking under the openai module. The original OpenAI module can coexist. 
We will still face this situation in the future, and my suggestion is to unify the underlying interfaces, such as llm and embedding, where calls to OpenAI are reused from the `ext.openai` module. 

I suggest that we can do this unification after completing the embedding module.

WDYT?

Reply:
Ok.